scripts/ci/qemu-run.sh

```diff
@@ -37,11 +37,13 @@ This is the QEMU wrapper script for the Dasharo Open Source Firmware Validation.
 
   Environmental variables:
     DIR         working directory, defaults to current working directory
+    DRIVE_PATH  path to file which will be added as a virtual hard drive
 
 Example usage:
     ./$(basename $0) vnc firmware
     ./$(basename $0) graphic os_install
     DIR=/my/work/dir ./$(basename $0) graphic os
+    DRIVE_PATH=dts.img ./$(basename $0) graphic firmware
```

Comment:
should it not be `graphic os`? Will it not boot our image? Also maybe not name that `dts.img` as we probably could use different images also

Reply:
Amended here 2c0e2eec2b029ea35cf8b95367bc91a79953118b.